Documentation/Introduction/Index.rst

```diff
@@ -53,8 +53,9 @@ Code examples
 Many of the code examples found in this document come from the TYPO3
 Core itself.
 
-Quite a few others come from the "examples" and the `styleguide <https://github.com/TYPO3/styleguide>`__ extension. 
-
+Quite a few others come from the "`styleguide <https://github.com/TYPO3/styleguide>`__" extension. You can
+install them if you want to try out these examples yourself and use them as
```

Comment:
Instead of `them` we should now use `it`.

Reply:
Right. Pushed again.